doc/release/release_notes_v1_2.md

```diff
@@ -0,0 +1,327 @@
+it to the RPC message before transferred over the network. Depending on the
+configuration, the DAOS Server may or may not calculate checksums to verify the
+data on receipt. On a fetch, the DAOS Server will send a known good checksum
+with the requested data to the DAOS Client, which will calculate checksums on
+the data received and verify.
+
+
+[More information](https://daos-stack.github.io/overview/data_integrity/)
+
+### Per-container ACL
+
+Client user and group access for containers is controlled by Access Control
+Lists (ACLs). Access-controlled container accesses include:
+
+ - Opening the container for access.
+ - Reading and writing data in the container.
+ - Reading and writing objects.
+ - Getting, setting, and listing user attributes.
+ - Getting, setting, and listing snapshots.
+ - Deleting the container (if the pool does not grant the user permission).
+ - Getting and setting container properties.
+ - Getting and modifying the container ACL.
+ - Modifying the container's owner.
+
+This is reflected in the set of supported [container permissions](https://daos-stack.github.io/overview/security/#permissions).
+
+[More information](https://daos-stack.github.io/user/container/#access-control-lists)
+
+### Improved Control Plane
+
+#### External Device States (via DMG)
+
+The device states that are returned from a device query by the administrator
+are dependent on both the persistently stored device state in SMD, and the
+in-memory BIO device list.
+
+ - NORMAL: A fully functional device in use by DAOS (or in setup).
+ - EVICTED: A device has been manually evicted and is no longer in use by DAOS.
+ - UNPLUGGED: A device previously used by DAOS is unplugged.
+ - NEW: A new device is available for use by DAOS.
+
+[More information](https://daos-stack.github.io/admin/administration/#nvme-ssd-health-monitoring)
+
+#### User Interfaces
+
+The DAOS dmg management utility can be used to query NVMe SSD health data, as
+well as provide useful commands for NVMe SSD eviction, hot plug, and device
+identification. NVMe hot plug and reintegration will not be fully supported
+until DAOS 2.2 release.
+
+ - Manually evict and NVMe SSD
+ - Replace an evicted SSD with a new NVMe SSD
+ - Reuse a faulty NVMe SSD (w/o reintegration)
+ - Locate a healthy NVMe SSD (for VMD-enabled devices only)
+ - Locate an evicted NVMe SSD (for VMD-enabled devices only)
+
+[More information](https://daos-stack.github.io/admin/administration/#nvme-ssd-eviction-and-hotplug)
+
+
+### Replication and Self Healing
+
+In DAOS, if the data is replicated with multiple copies on different targets,
+once one of the targets fail, the data on it will be rebuilt on the other
+targets. This reduces the data redundancy that would be impacted by the target
+failure.
+
+[More information]( https://daos-stack.github.io/admin/pool_operations/#target-exclusion-and-self-healing)
+
+
+### Offline Reintegration
+
+In DAOS, the excluded targets can be reintegrated back into the pool by dmg pool
```

Comment:
targets is confusing, what is meant here by target is server (I believe anyway).  Half the time it means SSD.